<robot name="bottle_fanta">
    <link name="bottle_fanta">
        <inertial>
            <origin xyz="0 0 0" rpy="0 0 0"/>
            <mass value="1"/>
            <inertia ixx="1" ixy="0" ixz="0" iyy="1" iyz="0" izz="1"/>
        </inertial>
        <visual>
            <origin xyz="0 0 0" rpy="0 0 0"/>
            <geometry>
                <mesh filename="package://tiago_bartender_world/meshes/bottle_fanta.dae"/>
            </geometry>
        </visual>
        <collision>
            <origin xyz="0 0 0" rpy="0 0 0"/>
            <geometry>
                <mesh filename="package://tiago_bartender_world/meshes/bottle_fanta.dae"/>
            </geometry>
        </collision>
    </link>
    <gazebo reference="bottle_fanta">
        <mu1>1.0</mu1>
        <mu2>1.0</mu2>
    </gazebo>
</robot>
